lua/damagelogs/shared/autoslay.lua

```diff
@@ -8,8 +8,15 @@ local function CreateCommand()
 	if mode != 1 and mode != 2 then return end
 	local aslay = mode == 1
 
+	Damagelog.markedForSlay = {}
 	function ulx.autoslay(calling_ply, target, rounds, reason)
-		Damagelog:SetSlays(calling_ply, target:SteamID(), rounds, reason, target)
+		Damagelog.markedForSlay[ target:SteamID() ] = {
+			admin = calling_ply,
```

Comment:
Does that mean the slay won't work if the admin disconnects before the round end ?

Reply:
Yeah maybe, instead of storing the player entity we could store their name instead.

Reply:
Better store the steam id and the name for displaying.